src/neuroconv/datainterfaces/behavior/medpc/medpcdatainterface.py

```diff
@@ -0,0 +1,248 @@
+from typing import Optional
+
+import numpy as np
+from pynwb.behavior import BehavioralEpochs, IntervalSeries
+from pynwb.file import NWBFile
+
+from neuroconv.basetemporalalignmentinterface import BaseTemporalAlignmentInterface
+from neuroconv.tools import get_package, nwb_helpers
+from neuroconv.utils import DeepDict, FilePathType
+
+from .medpc_helpers import read_medpc_file
+
+
+class MedPCInterface(BaseTemporalAlignmentInterface):
+    """Data Interface for MedPC output files.
```

Comment:
```suggestion
    """
    Data Interface for MedPC output files.
```

We tend to use this style on NeuroConv (when using `?` or `help` it also helps console clutter by adding a first layer of newline padding to the top)